### ISSUE TITLE
Illegal punning: 'is functional measurement of' and 'is measurement of'

### ISSUE THREAD

**Issue by jonathanbona** [addiehl/neuropsychological-testing-ontology#1]
When loading NPT.owl, Protege 5.2.0 reports the following errors:

```
Level:       ERROR
Time:        1554481495879
Message:     Illegal redeclarations of entities: reuse of entity http://purl.obolibrary.org/obo/NPT_0016000 in punning not allowed [Declaration(ObjectProperty(<http://purl.obolibrary.org/obo/NPT_0016000>)), Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/NPT_0016000>))]

Level:       ERROR
Time:        1554481495883
Message:     Illegal redeclarations of entities: reuse of entity http://purl.obolibrary.org/obo/NPT_0016001 in punning not allowed [Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/NPT_0016001>)), Declaration(ObjectProperty(<http://purl.obolibrary.org/obo/NPT_0016001>))]
```

The issue is that 'is functional measurement of' and 'is measurement of' are both declared as both annotation properties and object properties.

**Comment by APCox** [comment]
This error should be resolved (along with many others) in version 1.7.3.

**Comment by APCox** [comment]
Punning error was eliminated in version 1.7.3
